Raspi/Test/mqtt_wireless_log.py

```diff
@@ -154,7 +168,7 @@ def merge_temps(output_filename, temp_filepaths):
 
     # Merge the dataframes and output the final csv
     merged_dataframe = pd.concat(temp_dataframes, axis=1)
-    merged_dataframe.to_csv(output_filename)
+    merged_dataframe.to_csv("../logs/" + output_filename)
```

Comment:
It might be a good idea to clean up the temp files after you're done with them.

Reply:
Yeah they get cleaned up in the main body of the code